src/defines.ts

```diff
@@ -13,6 +21,7 @@ export type StatementType =
   | 'CREATE_TRIGGER'
   | 'CREATE_FUNCTION'
   | 'CREATE_INDEX'
+  | 'CREATE_PROCEDURE'
```

Comment:
It looks like should also be able to have `ALTER_PROCEDURE`. Would also be good to add `DROP_PROCEDURE` to run the gamut.

Reply:
Added on ee29727ce64e9fea82c4e8689952dc4996911273.